### PR TITLE
Provide Mock implementation of SampleSheet Component during Sample registration

### PR DESCRIPTION
**Pull request by Steffengreiner**
**What was changed** 
This PR achieves multiple things: 

1.) It provides a dummy implementation of the sample registration dialog containing the spreadsheet component and a general information layout (Meaning the provided data will not be read). 
2.) It showcases on how the tabsheet could be used as a stepper within the dialog to switch between views
3.) It provides mock implementations of a service providing the header values for the spreadsheet

### CODE REVIEW

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    dataTypeSelection.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
+    dataTypeSelection.setRenderer(new ComponentRenderer<>(MetaDataTypes -> {
+      Span metaDataType = new Span(MetaDataTypes.metaDataType);
+      Icon infoIcon = new Icon(VaadinIcon.INFO_CIRCLE);
+      infoIcon.addClassNames(IconSize.SMALL);
+      infoIcon.setColor("#77828f");
+      infoIcon.setTooltipText(MetaDataTypes.metaDataDescription);
+      return new HorizontalLayout(metaDataType, infoIcon);
+    }));
+  }
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java:86]: NIT: I don't like that the tooltip is not aligned, but next to the words.

Comment (wow-such-code) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java:86]: tooltip icon*

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleRegistrationDialog.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+  private void initTabStepper() {
+    tabStepper.add(generalInformationTab, generalInformationLayout);
+    tabStepper.add(sampleMetadataTab, sampleSpreadsheetLayout);
+    add(tabStepper);
+  }
+
+  private Tab createTabStep(String avatarLabel, String tabLabel) {
+    Avatar stepAvatar = new Avatar(avatarLabel);
+    stepAvatar.setColorIndex(2);
```
Comment (wow-such-code) [inline vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleRegistrationDialog.java:52]: what does this do? seems very hardcoded

**Review by wow-such-code — APPROVED** [review]
works for me, well done!

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleRegistrationDialog.java (hunk 2 above)
Comment (Steffengreiner) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleRegistrationDialog.java:52]: Sets the stepper color which is currently staying the same independent of selection

File: vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java (hunk 1 above)
Comment (Steffengreiner) [reply vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/GeneralInformationLayout.java:86]: Good point i'llm ake a note of it and fix it during the next iteration 👀

**Pull request merged**